### DISCUSSION TITLE
DirectML requirements?

### DISCUSSION THREAD

**Discussion by jonathancasper** [microsoft/onnxruntime-genai#387] [Q&A] [answered]
I'm trying to install the DirectML library with "pip install --pre onnxruntime-genai-directml" in WSL2, but I keep getting back an error: 

ERROR: Could not find a version that satisfies the requirement onnxruntime-genai-directml (from versions: none)
ERROR: No matching distribution found for onnxruntime-genai-directml

Is there something I'm missing about the requirements for the install?  I already have numpy installed, and my WSL2 instance is running Ubuntu 22.04 (with Python version 3.10.12).  I'm running an Intel processor with Iris XE (I understand if integrated graphics wasn't what you had in mind for this option, but it's still a GPU?).

Thanks!

**Comment by baijumeswani** [comment, marked as answer]
The `onnxruntime-genai-directml` wheels are only made available for windows. You can use `onnxruntime-genai` or `onnxruntime-genai-cuda` on linux.

**Reply by jonathancasper** [reply]
Ah, I see.  I guess I misunderstood the WSL support.  Thanks!
